fetch.bs

```diff
@@ -873,15 +873,15 @@ not always relevant and might require different behavior.
    <td>HTML's <code>&lt;script></code>, <code>importScripts()</code>
   <tr>
    <td>"<code>serviceworker</code>"
-   <td>?
+   <td><code>child-src</code>, <code>script-src</code>, <code>worker-src</code>
```

Comment:
`child-src` is ostensibly deprecated, but until any browser other than Chrome implements `worker-src` with the new inheritance structure, I guess you're right that it's worth keeping it in this list.